pom.xml

```diff
@@ -4,7 +4,7 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>com.odysseusinc.arachne</groupId>
     <artifactId>${final.artifactId}</artifactId>
-    <version>1.19.1</version>
+    <version>1.19.13</version>
```

Comment:
I don't think it is the right number here. If 1.19.1 was not published into the repository I think it should remain as is, otherwise it should be 1.19.2 I believe

Reply:
This is to ensure that reference can be updated in ArachneCentralAPI and keep it consistent on the master branch.
Just in case there will be a minor release from master. Backend change will break FE behaviour if unmatched by FE changes.